packages/shared/chat/sagas.ts

```diff
@@ -325,12 +327,17 @@ function initChatCommands() {
 
       getUnityInstance().TriggerSelfUserExpression(expression)
 
+      let body = ''
+      if (isFeatureEnabled(store.getState(), FeatureFlags.EMOTES_CUSTOMIZATION, false))
+        body =
+          '[WARNING] This command will be deprecated soon, take a look at our brand new emotes wheel by pressing B!'
```

Comment:
Why are we deprecating this? Operative systems create their UI but never remove the command line `cp` commands

Reply:
It's a product requirement.

Now users can equip up to 10 emotes on their profile, this allows us to control the memory since we dont have to load all the GLTFs containing the animations for people with hundreds of NFT Emotes. 
The command feature brings too little compared to the confussion caused by the fact that you can type `/emote wave` while not having the Wave emote equipped and thus nothing happening.